### PR TITLE
[No Ticket] Bump dependencies

### PR DESCRIPTION
**Pull request by broadbot**
Consolidate scala-steward PRs

### CODE REVIEW

**Review by rtitle — APPROVED** [review]

**Comment by rtitle** [comment]
jenkins retest

**Comment by rtitle** [comment]
jenkins retest

**Comment by rtitle** [comment]
jenkins retest

**Comment by rtitle** [comment]
Lot of flakiness in Jenkins jobs: leo-build, fiab-start…

jenkins retest

**Comment by rtitle** [comment]
jenkins retest

**Comment by rtitle** [comment]
jenkins retest

**Comment by rtitle** [comment]
jenkins retest

File: Dockerfile-tests [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM hseeberger/scala-sbt:graalvm-ce-21.0.0-java11_1.5.0_2.13.5
+FROM broadinstitute/scala-baseimage:jdk11-2.13.5-1.4.7
```
Comment (Qi77Qi) [inline Dockerfile-tests]: this probably doesn't need to be changed?

Comment (rtitle) [reply Dockerfile-tests]: Yeah agree.. wanted to see if it fixed tests hanging but seems not so will revert

**Pull request merged**